package Exceptionday;

public class Exceptionday6 {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		int a=10;
		int b=0;
		try {
		
			int c=a/b;
			System.out.println(c);
		}
		catch(ArithmeticException e) {
			System.out.println("cannot divide");
		}
		System.out.println("program completed");
	}

}
